diff --git a/src/base/kaldi-types.h b/src/base/kaldi-types.h
index 7ebf4f853..c15b288b2 100644
--- a/src/base/kaldi-types.h
+++ b/src/base/kaldi-types.h
@@ -41,6 +41,7 @@ typedef float   BaseFloat;
 
 // for discussion on what to do if you need compile kaldi
 // without OpenFST, see the bottom of this this file
+/*
 #include <fst/types.h>
 
 namespace kaldi {
@@ -53,10 +54,10 @@ namespace kaldi {
   typedef float   float32;
   typedef double double64;
 }  // end namespace kaldi
+*/
 
 // In a theoretical case you decide compile Kaldi without the OpenFST
 // comment the previous namespace statement and uncomment the following
-/*
 namespace kaldi {
   typedef int8_t   int8;
   typedef int16_t  int16;
@@ -70,6 +71,5 @@ namespace kaldi {
   typedef float    float32;
   typedef double   double64;
 }  // end namespace kaldi
-*/
 
 #endif  // KALDI_BASE_KALDI_TYPES_H_
diff --git a/src/feat/feature-functions.h b/src/feat/feature-functions.h
index 52454f304..a50322952 100644
--- a/src/feat/feature-functions.h
+++ b/src/feat/feature-functions.h
@@ -27,7 +27,7 @@
 #include <vector>
 
 #include "matrix/matrix-lib.h"
-#include "util/common-utils.h"
+// #include "util/common-utils.h"
 #include "base/kaldi-error.h"
 
 namespace kaldi {
diff --git a/src/feat/resample.cc b/src/feat/resample.cc
index 11f4c62bf..43ab8c962 100644
--- a/src/feat/resample.cc
+++ b/src/feat/resample.cc
@@ -23,7 +23,7 @@
 
 #include <algorithm>
 #include <limits>
-#include "feat/feature-functions.h"
+// #include "feat/feature-functions.h"
 #include "matrix/matrix-functions.h"
 #include "feat/resample.h"
 
diff --git a/src/feat/resample.h b/src/feat/resample.h
index e0b4688c9..43fcf3fa4 100644
--- a/src/feat/resample.h
+++ b/src/feat/resample.h
@@ -31,7 +31,7 @@
 
 
 #include "matrix/matrix-lib.h"
-#include "util/common-utils.h"
+// #include "util/common-utils.h"
 #include "base/kaldi-error.h"
 
 namespace kaldi {
diff --git a/src/matrix/matrix-lib.h b/src/matrix/matrix-lib.h
index b6059b06c..4fb9e1b16 100644
--- a/src/matrix/matrix-lib.h
+++ b/src/matrix/matrix-lib.h
@@ -25,14 +25,14 @@
 #include "base/kaldi-common.h"
 #include "matrix/kaldi-vector.h"
 #include "matrix/kaldi-matrix.h"
-#include "matrix/sp-matrix.h"
-#include "matrix/tp-matrix.h"
+// #include "matrix/sp-matrix.h"
+// #include "matrix/tp-matrix.h"
 #include "matrix/matrix-functions.h"
 #include "matrix/srfft.h"
 #include "matrix/compressed-matrix.h"
-#include "matrix/sparse-matrix.h"
+// #include "matrix/sparse-matrix.h"
 #include "matrix/optimization.h"
-#include "matrix/numpy-array.h"
+// #include "matrix/numpy-array.h"
 
 #endif
 
